src/style.css

```
@@ -0,0 +1,102 @@
+@import url(./default.css);
+
+@import url(./stars.css);
+
+@import url(./var.css);
```

file with vars should be imported in the first queue
```suggestion
@import url(./default.css);
@import url(./var.css);

@import url(./stars.css);

```